src/pages/VenuePage/VenuePage.tsx

```diff
@@ -81,6 +85,7 @@ export const VenuePage: React.FC<VenuePageProps> = ({
   user,
   profile,
   userLocation,
+  setBackButtonTargetSpace,
```

Comment:
`backSpace`? Sounds funny, but it's shorter 🤷

Reply:
`setBackButtonSpace` in 22c9bf0fd